src/kernels/level1/xamax.opencl

```diff
@@ -30,7 +30,11 @@ R"(
 // =================================================================================================
 
 // The main reduction kernel, performing the loading and the majority of the operation
-__kernel __attribute__((reqd_work_group_size(WGS1, 1, 1)))
+#if RELAX_WORKGROUP_SIZE == 1
+  __kernel
+#elif
```

Comment:
Not sure what the difference is between an empty `#elif` and a normal `#else`, but the latter reads a bit nicer I think. Could you easily change all those `#elif`'s in `#else`'s?

Reply:
Typo. `#elif` has been changed to `#else` in all the kernel files.